src/com/lftechnology/java/training/srijan/FirstTutorial.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java.training.srijan;
+
+import java.util.Scanner;
+
+public class FirstTutorial {
+public static void main(String[] args){
+		
+		Scanner input = null;
+		
+		try{
+			input = new Scanner(System.in);
+			System.out.println("Enter your Name with Salutation");
+			String nameWithSalutation = input.nextLine();
+			nameWithSalutation=nameWithSalutation.trim();
+			System.out.printf("%n%s%n", nameWithSalutation);
+			
+			String[] text = nameWithSalutation.split(" ");
+			System.out.println("Salutation:" + text[0]);
+			//System.out.println("Name:" + text[1] +" "+ text[2]);
+			System.out.println("Name:");
```

Comment:
We never use println in real application, make habit of using logger. There are many third party in java like log4j, slf4j. You should be aware of synchronous vs asynchronous logger. 
http://www.mkyong.com/tutorials/log4j-tutorial/
http://stackoverflow.com/questions/5950557/good-examples-using-java-util-logging